exporter/signalfxexporter/translation/constants.go

```diff
@@ -72,6 +72,9 @@ translation_rules:
     container.memory.rss: container_memory_rss_bytes
     container.memory.usage: container_memory_usage_bytes
     k8s.node.cpu.utilization: cpu.utilization
+    container.memory.working_set: container_memory_working_set_bytes
+    container.memory.page_faults: container_memory_page_faults
+    container.memory.major_page_faults: container_memory_major_page_faults
```

Comment:
Could you move it a bit up to keep this list sorted

Reply:
Done.